src/parser/shaman/enhancement/modules/Abilities.tsx

```diff
@@ -47,15 +47,6 @@ class Abilities extends CoreAbilities {
         gcd: {
           base: 1500,
         },
-        castEfficiency: {
```

Comment:
What are you trying to accomplish?

Do you want to remove it from the list of spells at the bottom of the Statistics tab? Or are you just trying to remove the Suggestion that says you arent casting it enough?

Reply:
This particular change is to remove it from the spell suggestions, as it should no longer be considered as an option for an offensive GCD.

Reply:
then just change suggestion: true to suggestion: false and get rid of recommendedEfficiency and extraSuggestion

Reply:
Got it, will do.